Makefile

```diff
@@ -53,18 +53,26 @@ docker-build:
 
 # Run HTTP server locally
 run-http:
-	./$(BINARY_NAME) http --port 8080
+	./$(BINARY_NAME) http --transport-port 8080 --transport-host 0.0.0.0
```

Comment:
I'll close the other PR? @wellsiau-aws  seems like you need this to proceed

Reply:
sure, let me add ideas from #107 here.

at high-level , default host bind is 127.0.0.1 unless specified.